### DISCUSSION TITLE
Versão do Postgres

### DISCUSSION THREAD

**Discussion by c13c83** [cleydyr/biblivre#346] [Q&A] [answered]
Olá, 

Tem alguma restrição em usar a versão 12 do postgres como biblivre, pois na instação está sendo usado o 9, mas preciso instalar em um servidor que já está rodando. 

Obrigado

**Comment by cleydyr** [comment, marked as answer]
Cara, tu acredita que eu só vim ver essa funcionalidade de discussão hoje? 😅 

Então, a essa altura você já deve ter se aventurado e visto que não tem. A última versão oficial do Biblivre usava o PostgreSQL 9.1. Acontece que as versões do PostgreSQL são bastante retrocompatíveis, ou seja, tudo que funciona numa versão anterior deve funcionar numa versão mais nova. O contrário já não é verdade: se você está trabalhando com um backup do Biblivre conectado ao PostgreSQL 12 e tenta importar em outra instância do Biblivre conectado à versão 9.1, por exemplo, a importação vai falhar.

Eu desconheço qualquer incompatibilidade entre a versão 12 e a 9 que afete o Biblivre.

Espero que ajude.
